README.md

```diff
@@ -13,7 +13,7 @@ Backstage is an open platform for building developer portals.
 
 The philosophy behind Backstage is simple: Don't expose your engineers to the full complexity of your infrastructure tooling. Engineers should be shipping code — not figuring out a whole new toolset every time they want to implement the basics. Backstage allows you add "stuff" (tooling, services, features, etc.) by adding a plugin, instead of building a new tool. This saves you work and avoids the need of your team to learn how to use and support yet another tool.
 
-For more information go to [backstage.io](https://backstage.io)
+For more information go to [backstage.io](https://backstage.io) or join our [Discord chatroom](https://discord.gg/EBHEGzX).
```

Comment:
i was wondering about the use of a single word "chatroom" as opposed to "chat room", but it seems it's pretty widespread :)